### DISCUSSION TITLE
右側のLEDがつかない

### DISCUSSION THREAD

**Discussion by KohsukeIde** [Yowkees/keyball#351] [Q&A]
左右両方に以下のファーム ウェアを flash https://github.com/kamiichi99/keyball/tree/main/qmk_firmware/keyboards/keyball/keyball39/keymaps/kamidai

ファームウェア的にはトラックボールを操作している間は左右両方のLEDが白く光るべき。

- case 1
どちらをマスターにしても右側はLEDが白く光らないが、操作には問題がない。

- case 2
たまに光るが、ファームウェアに基づいた真っ白のバックライトではなく、トラックボールを操作していない時も光ったまま。おそらく、ファームウェアを入れる前に使ってた色になっている気がする。 この状態になると、トラックボールを動かしても反応がない。（一瞬だけカーソルが動いてレイヤーが変わることはあるが、その後は動かなくなる。）

追記:
テストファームウェアの時から右側のLEDの調子がおかしいことは何回かあった。以下の正規ファームウェアを入れたときも右側だけLEDがつかない問題が頻繁にあった。
https://remap-keys.app/catalog/k895xiCpsM5zlYZCPTEN/firmware のKeyball39_via

右側のキー入力には問題はないです。LEDが光る時は全てのLEDが光るので、はんだの問題でもないとおもっています。

これらの情報から何が問題かわかったりしますでしょうか？

**Comment by Yowkees** [comment]
大変申し訳ありませんが、私はリンク先のファームウェアを追い切れていませんので、本件はFlashしたファームウェアの制作者様へお問合せいただいた方が確実かと思います。
お手数ですがよろしくお願いいたします。
